readme.html

```diff
@@ -29,6 +29,31 @@
 			   <li>
 					max_container: Maximum containers to ingest for subsequent runs of scheduled polling.
 			   </li>
+			   <li>
+				   severity: Custom severity of the ingested artifact.
+				</li>
+				<br>
+				<ul>
+					<li>
+				   		<b>Using Custom Severities on Ingested Data</b>
+						<ul>
+						<li>
+							If you decide to use custom severities to apply to Containers and Artifacts ingested via On Poll, then you must make sure that the automation user you use has the correct permissions.
+						</li>
+						<li>
+							By default, the automation user is selected to run the ServiceNow ingestion action. (See <b>Asset Configuration</b>><b>Asset Settings</b>><b>Advanced</b>)
+							The automation user does <b>NOT</b> have access to view or edit <b>System Settings</b>, which includes the permission to view the custom severities on your instance.
+							This will cause your On Poll action to fail since your user cannot pull the custom severities on your instance.
+						</li>
+						<li>
+							In order to solve this problem, you must create a user of type <b>Automation</b> and assign this user a Role that has permissions to view or edit <b>System Settings</b>
```

Comment:
Would it be possible to include an example here?
Showing steps / screenshots here should greatly reduce future support tickets if they run into issues here.

Reply:
Yes I have made the suggested changes.